bundles/SeoBundle/config/maintenance.yaml

```diff
@@ -0,0 +1,16 @@
+services:
+    _defaults:
+        public: false
+        autowire: true
+        autoconfigure: true
+
+    #
+    # Maintenance
+    #
+    # Configure all maintenance tasks
+
+    Pimcore\Bundle\SeoBundle\Maintenance\RedirectCleanupTask:
```

Comment:
Can we please add the HTTP Cleanup task here too, to be more consistent? 
Should be in the services.yaml atm.